content/vault/v1.20.x/content/docs/secrets/kv/kv-v2/cookbook/patch-data.mdx

```diff
@@ -22,6 +22,8 @@ an existing data path in the `kv` v2 plugin.
 
 </Tip>
 
+Note: if the secret does not already exist, the patch will not create it.
```

Comment:
```suggestion
- The keys or subkeys already exist with editable values. Patching does
   not create new secrets.
```

Instead of having a floating sentence, we should add this to the list after "You know the keys or subkeys you want to patch".